xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -119,27 +125,52 @@ export default {
   // The methods for specific displayers can be found in the displayerMixin
   methods: {
 
+    toggleIsView() {
+      this.$emit('update:isView', !this.isView);
+    },
+
+    setView() {
+      this.$emit('update:isView', true);
+    },
+    
+    setEdit() {
+      this.$emit('update:isView', false);
+    },
+
     // Trigger View mode (switch from Editor widget to Viewer widget)
     // This should rarely be used directly as it does not validate modified data
     // Used the `applyEdit` method instead (found in the displayerMixin)
     // which call this view function after validating data
-    view () {
-      if (this.isView) { return; }
-      this.isView = true;
+    view() {
```

Comment:
Why would you call ``view`` if not to switch to view mode? You removed the line that was setting ``isView``.

Reply:
This is still an issue. The function name along with its documentation are very misleading. You have a function named ``view`` that doesn't do anything on view mode and focuses the element on edit / loading mode?